documentation/general/eliminate-duplicate-file.md

```diff
@@ -0,0 +1,433 @@
+A 50 MB reduction in compressed size translates to 37.5 TB per week in bandwidth saved from official .NET container pulls alone.
+Beyond containers, SDK installations also happen at high volume in CI/CD pipelines, cloud development environments, and through various tooling extensions.
+
+The [Native AOT SDK epic](https://github.com/dotnet/sdk/issues/40931) represents a significant investment in improving performance, but it comes at the cost of increased SDK size.
+Without addressing duplication, Native AOT will exacerbate the size problem and likely make customers in CI and container scenarios unhappy.
+This deduplication work gives us the headroom needed to absorb the Native AOT expansion while still delivering a better overall experience.
+
+## By the Numbers
+
+A duplicate file analysis was performed on the SDK layout of .NET 10.0.100, where file uniqueness is defined by *name*, *TFM* (Target Framework Moniker), and *culture*.
+The following data is the result of this analysis using the [SdkLayoutAnalyzer](https://github.com/MichaelSimons/SdkLayoutAnalyzer) tool.
+
+The goal is for the vast majority if not all of the components within the SDK to depend on and use the same version of their dependencies.
+
+**Notes:**
+1. The baseline measurements in the following tables compare the complete .NET installation (including runtimes, packs, shared frameworks, host, etc.) versus just the SDK directory contents.
+1. Standard archive formats (zip and tar) do not automatically deduplicate identical files, each file is compressed independently, even when multiple files have identical content.
+This means that duplicate files consume space both on disk after extraction and within the compressed archive itself.
+
+### Linux x64
+
+| Metric       | Baseline 10.0 (dotnet / SDK only) | Detected SDK Duplicates | % Duplicates (dotnet / SDK only) |
+|--------------|-----------------------------------|-------------------------|----------------------------------|
+| Archive Size | 229 MB / 150 MB                   | 53 MB                   | 23.1% / 35.3%                    |
+| Disk Size    | 623 MB / 402 MB                   | 140 MB                  | 22.5% / 34.8%                    |
+| File Count   | 4,915 / 3,667                     | 816 files               | 16.6% / 22.2%                    |
+
+### Windows x64
+
+| Metric       | Baseline 10.0 (dotnet / SDK only) | Detected SDK Duplicates | % Duplicates (dotnet / SDK only) |
+|--------------|-----------------------------------|-------------------------|----------------------------------|
+| Disk Size    | 789 MB / 411 MB                   | 148 MB                  | 18.8% / 36.0%                    |
+| File Count   | 5,610 / 3,791                     | 908 files               | 16.2% / 24.0%                    |
+
+**Note:** The WindowsDesktop runtime and target pack, along with .NET Framework support are the major reasons for the differences between Windows and Linux.
```

Comment:
Might be useful for comparisons sake to include the sizes the windowsdesktop runtime and framework support.